src/logger.ts

```diff
@@ -0,0 +1,37 @@
+import pino from 'pino';
+import { hostname } from 'os';
+import { version } from '../package.json';
+
+const level = (() => {
```

Comment:
It might be one liner:
```
const level = ['test', 'development'].includes(process.env.NODE_ENV) ? 'trace' : 'info';
```

Reply:
+1, switch is overkill for most cases. It does make sense for hmm 3-4+ cases?